src/seedu/addressbook/AddressBook.java

```diff
@@ -136,14 +129,11 @@
     private static final String DIVIDER = "===================================================";
 
 
-    /* We use a String array to store details of a single person.
-     * The constants given below are the indexes for the different data elements of a person
-     * used by the internal String[] storage format.
-     * For example, a person's name is stored as the 0th element in the array.
+    /* We use a HashMap to store details of a single person. Use enum to access fields of a person.
+     *     // usage HashMap<PersonProperty,String> john = new HashMap<>();
```

Comment:
here should follow the previous message `For example, ... ` instead of double // to keep things standardized